unix/qstrdefsport.h

```diff
@@ -79,6 +79,7 @@ Q(listen)
 Q(accept)
 Q(recv)
 Q(recvfrom)
+Q(settimeout)
```

Comment:
So the pre-processor does multiple passes? First, on this header file and then on the generated header file? 

What's the harm of adding this line? The way I see it, it allows to compile using the lwIP lib and many of the other lines in this file are (correct me if I'm wrong) not needed; I would assume that all unnecessary generated header files will be optimized away regardless.